keycloak/README.md

```diff
@@ -52,6 +52,7 @@ Linux command
     **Note**: 
       - `PORT`: Port should be similar to the port provided in **Configure syslog message forwarding from keycloak** section.
       - It is recommended not to change the service and source values, as these parameters are integral to the pipeline's operation.
+      - In Keycloak version 26.3, [counting framing] [8] for log messages was introduced. This can cause the log message to not be parsed correctly in the log pipeline. This feature can be disabled by setting `log-syslog-counting-framing=false` in the Keycloak configuration file. 
```

Comment:
```suggestion
      - In Keycloak version 26.3, [counting framing][8] for log messages was introduced. This can cause the log message to not be parsed correctly in the log pipeline. This feature can be disabled by setting `log-syslog-counting-framing=false` in the Keycloak configuration file. 
```